src/Card/Card.tsx

```diff
@@ -0,0 +1,191 @@
+/** @jsx jsx */
+import { colors } from "../colors";
+import { base } from "../typography";
+import { jsx } from "@emotion/core";
+import React from "react";
+import PropTypes from "prop-types";
+
+const descriptionMaxWidth = 650;
+
+interface Props
+  extends React.DetailedHTMLProps<
+    React.HTMLAttributes<HTMLDivElement>,
+    HTMLDivElement
+  > {
+  /**
+   * The content of the card,
+   * appears below the title and description
+   */
+  children?: React.ReactNode;
+
+  heading?: React.ReactNode;
+
+  /**
+   * the description for this card
+   * appears in grey below the title
+   */
+  description?: React.ReactNode;
+
+  /**
+   * actions could be a button
+   * or a tooltip or anything the card should display
+   * aligned with the title on the right
+   */
+  actions?: React.ReactNode;
+
+  size?: "standard" | "large";
```

Comment:
Maybe a comment here about how this increases the padding and heading font size would be nice.